instrument-starters/opentracing-spring-cloud-redis-starter/src/main/java/io/opentracing/contrib/spring/cloud/redis/connection/RedisCommand.java

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright 2017-2018 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.cloud.redis.connection;
+
+/**
+ * A list of Redis comands.
+ *
+ * Redis command list (some commands which are not in this page but they exists in the Java API were added):
+ * https://redis.io/commands
+ *
+ * @author Daniel del Castillo
+ */
+final class RedisCommand {
```

Comment:
could be this names consumed from redis lib? we could also use class names

Reply:
do you mean from `org.springframework.data.redis.core.RedisCommand`? I couldn't find all the operations in this enum, e.g. cluster operations are not defined there. I didn't look in specific client libraries.